test/test_filters.rb

```diff
@@ -57,6 +57,20 @@ def select; end
       )
     end
 
+    should "markdownify with simple string in inline mode" do
+      assert_equal(
+        "one <em>simple</em> string",
+        @filter.markdownify("one _simple_ string", :inline)
+      )
+    end
+
+    should "markdownify with multi-line string in inline mode" do
+      assert_equal(
+        "first line\nsecond line",
+        @filter.markdownify("first line\n\nsecond line", :inline)
```

Comment:
`one\n\ntwo` -> `<p>one</p>\n<p>two</p>` (stripped `p` tags) -> `one\ntwo`... this shouldn't be happening.